alea/models/blueice_extended_model.py

```diff
@@ -272,8 +281,9 @@ def store_data(self, file_name, data_list, data_name_list=None, metadata=None):
 
     def _generate_science_data(self, **generate_values) -> dict:
         """Generate the science data for all likelihood terms except the ancillary likelihood."""
+        livetime_days = [generate_values.get(ln,None) for ln in self.livetime_parameter_names]
         science_data = [
-            gen.simulate(**generate_values) for gen in self.data_generators]
+            gen.simulate(livetime_days = lt, **generate_values) for gen, lt in zip(self.data_generators, livetime_days)]
```

Comment:
Maybe change `(ln,None)` to `(ln, None)`, and `livetime_days = lt,` to `livetime_days=lt,`.